moveit_planners/ompl/package.xml

```diff
@@ -34,6 +34,7 @@
   <test_depend>tf2_eigen</test_depend>
   <buildtool_depend>eigen3_cmake_module</buildtool_depend>
   <test_depend>ament_cmake_gtest</test_depend>
+  <test_depend>ros_testing</test_depend>
```

Comment:
Looking at https://github.com/ros-planning/moveit2/blob/main/moveit_planners/ompl/ompl_interface/CMakeLists.txt we are not using any of `ros_testing` functionality, I think we should remove https://github.com/ros-planning/moveit2/blob/main/moveit_planners/ompl/ompl_interface/CMakeLists.txt#L68 rather than adding it here

```suggestion
```

Reply:
done.

Reply:
still there?

Reply:
uhm, no?

Reply:
... I confused the files